### ISSUE TITLE
unet file code is not getting images from given path

### ISSUE THREAD

**Issue by romaanmirza** [theidentity/Unets#2]
i am trying to run this unet code with given paths.. i am not change anything and everything is according to requirement... but this is not working... no image is loaded.. and error is:

"E:\BRL\liver\Unets-master\python programs\Scripts\python.exe" E:/BRL/liver/Unets-master/unet.py
Using TensorFlow backend.
E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\framework\dtypes.py:526: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\framework\dtypes.py:527: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\framework\dtypes.py:528: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\framework\dtypes.py:529: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\framework\dtypes.py:530: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\framework\dtypes.py:535: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
WARNING:tensorflow:From E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\ops\resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
Found 0 images belonging to 1 classes.
Found 0 images belonging to 1 classes.
Found 0 images belonging to 1 classes.
Found 0 images belonging to 1 classes.
WARNING:tensorflow:From E:\BRL\liver\Unets-master\python programs\lib\site-packages\tensorflow\python\ops\math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Epoch 1/20
2019-11-28 12:34:30.907344: W tensorflow/core/framework/allocator.cc:124] Allocation of 301989888 exceeds 10% of system memory.

 1/28 [>.............................] - ETA: 48s - loss: nan - accuracy: nan2019-11-28 12:34:31.310869: W tensorflow/core/framework/allocator.cc:124] Allocation of 301989888 exceeds 10% of system memory.

 2/28 [=>............................] - ETA: 28s - loss: nan - accuracy: nan2019-11-28 12:34:31.727624: W tensorflow/core/framework/allocator.cc:124] Allocation of 301989888 exceeds 10% of system memory.

 3/28 [==>...........................] - ETA: 21s - loss: nan - accuracy: nan2019-11-28 12:34:32.119705: W tensorflow/core/framework/allocator.cc:124] Allocation of 301989888 exceeds 10% of system memory.

 4/28 [===>..........................] - ETA: 18s - loss: nan - accuracy: nan2019-11-28 12:34:32.516377: W tensorflow/core/framework/allocator.cc:124] Allocation of 301989888 exceeds 10% of system memory.

 5/28 [====>.........................] - ETA: 15s - loss: nan - accuracy: nan
 6/28 [=====>........................] - ETA: 13s - loss: nan - accuracy: nan
 7/28 [======>.......................] - ETA: 12s - loss: nan - accuracy: nan
 8/28 [=======>......................] - ETA: 11s - loss: nan - accuracy: nan
 9/28 [========>.....................] - ETA: 10s - loss: nan - accuracy: nan
10/28 [=========>....................] - ETA: 9s - loss: nan - accuracy: nan 
11/28 [==========>...................] - ETA: 8s - loss: nan - accuracy: nan
12/28 [===========>..................] - ETA: 8s - loss: nan - accuracy: nan
13/28 [============>.................] - ETA: 7s - loss: nan - accuracy: nan
14/28 [==============>...............] - ETA: 7s - loss: nan - accuracy: nan
15/28 [===============>..............] - ETA: 6s - loss: nan - accuracy: nan
16/28 [================>.............] - ETA: 5s - loss: nan - accuracy: nan
17/28 [=================>............] - ETA: 5s - loss: nan - accuracy: nan
18/28 [==================>...........] - ETA: 4s - loss: nan - accuracy: nan
19/28 [===================>..........] - ETA: 4s - loss: nan - accuracy: nan
20/28 [====================>.........] - ETA: 3s - loss: nan - accuracy: nan
21/28 [=====================>........] - ETA: 3s - loss: nan - accuracy: nan
22/28 [======================>.......] - ETA: 2s - loss: nan - accuracy: nan
23/28 [=======================>......] - ETA: 2s - loss: nan - accuracy: nan
24/28 [========================>.....] - ETA: 1s - loss: nan - accuracy: nan
25/28 [=========================>....] - ETA: 1s - loss: nan - accuracy: nan
26/28 [==========================>...] - ETA: 0s - loss: nan - accuracy: nan
27/28 [===========================>..] - ETA: 0s - loss: nan - accuracy: nan
E:\BRL\liver\Unets-master\python programs\lib\site-packages\keras\callbacks\callbacks.py:820: RuntimeWarning: invalid value encountered in less
28/28 [==============================] - 13s 467ms/step - loss: nan - accuracy: nan - val_loss: nan - val_accuracy: nan

  if self.monitor_op(current - self.min_delta, self.best):
Epoch 00001: val_loss did not improve from inf
E:\BRL\liver\Unets-master\python programs\lib\site-packages\keras\callbacks\callbacks.py:709: RuntimeWarning: invalid value encountered in less
  if self.monitor_op(current, self.best):
Epoch 2/20

 1/28 [>.............................] - ETA: 13s - loss: nan - accuracy: nan
 2/28 [=>............................] - ETA: 11s - loss: nan - accuracy: nan
 3/28 [==>...........................] - ETA: 10s - loss: nan - accuracy: nan
Process finished with exit code -1

**Comment by theidentity** [comment]
Most probably, it is because I had written the script for use in Linux. If you are using windows, try replacing the forward slashes to backward slashes. That should solve the issue.

eg : data/all/images_mtgm/1.png  to data\all\images_mtgm\1.png

**Comment by romaanmirza** [comment]
thanks
